external/test-runtime/optional.json

```diff
@@ -4,7 +4,9 @@
       "dependencies": {
         "Microsoft.DotNet.IBCMerge": "4.6.0-alpha-00001",
         "Microsoft.DotNet.UAP.TestTools": "1.0.2",
-        "TestILCNugetPackageForCoreFX": "1.0.0-beta-25304-01"
+        "TestILC.amd64ret": "1.0.0-beta-25305-00",
```

Comment:
> @mellinoe do you have an issue tracking removing that one as well?

I don't. As far as I know, we can't move this off of project.json currently. It uses some .NET Framework-only task for "encrypted nuget restore" or something like that. It isn't supported by the x-plat CLI.